### ISSUE TITLE
Adjusting policy on [Root Language] and [Root Form]

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/CoBL#274]
After various discussions in person, by email and through github issues, and noting also the wiki page on [Root Forms and Root Languages](https://github.com/lingdb/CoBL/wiki/06-Root-Forms-and-Root-Languages), it is clear that we need to adjust the policy on the 'Root Language' and 'Root Form' fields that are the basic, quick way of identifying a cognate set (other tand looking at all its member lexemes, of course).

For the cognate sets on which most attention has rightly been focused so far, those that go back to Proto-Indo European itself, (or at least, are found in more than one primary branch of Indo-European) nothing changes.  Proto-Indo European remains the 'Root Language', and the 'Root Form' remains the PIE reconstruction.

For cognate sets that are found only in a single primary branch of the family in the CoBL database, the policy so far has been as follows.
- Case 1.  A cognate set is limited to a single branch _in our CoBL data set_ (because it is only found in one branch in one of our reference meanings), **but** is securely known to go back to Proto-Indo European, albeit with different meanings in other branches.  (The qualification 'securely' is important here;  there are many speculative associations between particular branches for particular roots, but take only those that it is widely and securely agreed can be reconstructed back to PIE.)  In this case, the earliest known form of the root (on which the cognacy is based) does indeed go back to Proto-Indo European, which should therefore be the 'Root Language'.  
- Case 2a.  A cognate set is limited to a single branch in CoBL and is **not** otherwise known elsewhere in Indo-European.  Within that branch, it securely goes back to the proto-language for that branch.  In this case, the 'Root Language' is the proto-language for that branch, and the 'Root Form' is the reconstruction in that proto-language.  
- Case 2b.  A cognate set is limited to a single branch in CoBL and is **not** otherwise known elsewhere in Indo-European.  Within that branch, it does **not** securely go all the way back to the proto-language for that branch, but is only found sporadically in a few languages.  In this case, the 'Root Language' is just the branch name, and the 'Root Form' is somewhat unclear, but should preferably be given as the expected form it would have taken in that proto-language. 
- Case 3.  A cognate set is limited to an individual language (or set of very closely related dialects of it), and is generally considered definitely **not** to go back to the branch proto-language.  In this case, the 'Root Language' is just the language name, and the 'Root Form' is the earliest known form in that language.  

The reason for an adjustment is that in several crucial branches, there is relative uncertainty or debate surrounding reconstructions in the proto-language for that branch.  It can be very time-consuming, and still end up controversial, to research the latest debate on precise reconstructed forms for that proto-language stage.  

Meanwhile, many branches include ancient written languages which are not quite the direct ancestors of the modern spoken languages, but do on the other hand provide rich, unquestionable attestation of early forms of these cognate sets in a particular branch.  For practical CoBL purposes, this advantage in fact far outweighs the 'problem' that these languages are not the exact proto-languages of their respective branches.  The languages in question include notably, of course:  Vedic, Avestan, Classical Latin, Ancient Greek, Old Church Slavonic and (for the Goidelic sub-branch), Old Irish.

The adjustment, then, is that for branches with such a well-attested ancient language that is at least a **near-direct** ancestor, then its attested forms are actually often preferable as the reference form by which a cognate set can be easily recognised during data entry.  That is, in fact what CoBL needs is not strictly the absolute source form (on which there may be doubts on precise details) but just something highly recognisable and secure, as a reference form for that cognate set.  

So the adjustment includes even renaming these fields.  Quite what to call them (and to keep the name short enough for the webpage column header) I’m not sure, but at least to signal it, I suggest for now (until better suggestions):
`Source Language` should become `Source Ref. Lg.`
`Source Form` should become `Source Ref. Form`

Branch specialists should obviously feel free to stick to the original system wherever they prefer and feel confident to do so.  That will depend on many variables, but one can imagine cases where, for example, there is wide agreement and high confidence in a Proto-Slavic reconstruction, and the Old Church Slavonic is somewhat different.  In that case, it is perfectly fine to enter the 'Root Language' as Proto-Slavic, with the corresponding reconstruction, rather than the OCS form.

Just to explain more fully why this adjustment is really more appropriate for CoBL's purposes, it is basically because CoBL is not itself a work of reconstruction, of course.  It is a database of cognacy relationships across lexemes for 200 meanings.  That draws on reconstruction work over the last two centuries, richly documented in the literature, but CoBL does not aspire to add to that reconstruction work, just to make use of it to ensure that the database is as coherent as possible. 

CoBL should therefore generally try to avoid taking a strong position on reconstructions, when uncertain or debated.  For CoBL's purposes, the point of the 'Root Form' field for a cognate set is not to stake a claim in favour of one reconstruction or another, for a given proto-language.  It is a much simpler, practical aim:  to enable users to quickly recognise and identify the cognate set in question, when it appears displayed in CoBL pages and indeed the drop-down box for assigining cognacy to a given lexeme.

**Comment by m-scarborough** [comment]
This seems quite sensible.  I've begun working on the anonymous cognate sets within Hellenic this evening, and this policy seems to give a straightforward system for assigning [Root Language] and [Root Form] to the majority of cases.

I would, however, like to ask for one point of clarification.  Classes that are defined as loan-events – should these be assigned [Root Language] and [Root Form] at all?  I ask, because at some point someone did this for HOUSE σπίτι (a Latin loanword), and the input of the Latin borrowed form for [Root Form] and 'Greek' for [Root Language] alongside looks completely bizarre to me:

<img width="1102" alt="screen shot 2016-08-07 at 21 40 39" src="https://cloud.githubusercontent.com/assets/15671787/17465145/b070770e-5ce7-11e6-945b-91e55896bba5.png">

What shall be the policy for these cases?

**Comment by CasFre** [comment]
Thank you for drawing this up, @PaulHeggarty.

I think this will provide good guidance for further cognacy coding, but some modifications might still be needed.

- **A:** I think that the adjustment that you describe also pertains to cases of 2a, namely those where we do think it is safe to assume that the cognate set goes back to the branch proto-language, but we just don't know the exact reconstruction for that stage rightaway and can't spend the time investigating into that. I think that is the sort of cases we are coming from with the discussion under https://github.com/lingdb/CoBL/issues/264 and what your description of the adjustment fits best actually.

So, as a rule of thumb for **our overall policy**, covering at least 1, 2a and 2b (and with some minor additional assumptions also 3), one might say:
If reconstruction is securely possible,
then give the earliest safe _source_ form + language (_source_ meaning ancestral to all the different outcomes);
if this is unknown, then give the form and name of the _reference_ language (the _reference_ language being that one defined as such for the respective clade in the languages list, or subsidiary the one first in the language list for the respective clade which is represented in the cognate set).

If reconstruction is not securely possible or not possible at all,
then give again the form and name of the reference language (the reference language being that one defined as such for the respective clade in the languages list, or subsidiary the one first in the language list for the respective clade which is represented in the cognate set).


- **B:** As far as terminology is concerned, I would thus suggest simply _reference language_ as a new label instead of _source reference language_. My view would be that _reference language_ is a term that makes clear that, as @PaulHeggarty rightly stated, CoBL does not in the first place make its own claims on reconstruction and that it is itself just a handy label for identification. It may thus be a _source_ language, i.e. a common ancestor to all the languages represented in a cognate set, but it need not be. 
Actually, the term _reference language_ already exists in the database and does not need to be newly introduced, but just exploited further and be accompanied by _reference form_.
_Reference form_ and _source form_ would represent the same distinction mutatis mutandis.

- **C:** For the loan word cognate sets I would still say that it is advisable to have separate fields where one can enter source form and source language, although the ancestry in the first place came about by horizontal transfer, so to say.
The problem with the spiti-case that @m-scarborough mentioned is simply that something has gone wrong during data entry: "hospitium" as the form in source language Latin should have been entered in the field "form in loan source language" which I marked with a red arrow in the following screen shot:
![- hospitium](https://cloud.githubusercontent.com/assets/17431124/17475504/9bf94f70-5d5b-11e6-8e2a-e9384f2ca436.png)
According to the rules for deriving a reference language that I described and the information given on the individual cognate set page http://www.cobl.info/cognate/5170/, the entry for this specific cognate set for "root language" + "root form" / "reference language" and "reference form" would be "Greek: Modern Std" and "σπίτι", which seems to be acceptable.

**Comment by CasFre** [comment]
By the way, what's a good "root form" + "root language" or whatever we are going to call it from now on for a parallel loan or possible parallel loan? Cf. http://www.cobl.info/cognate/5711/ as a nice example.

**Comment by cormacanderson** [comment]
If the parallel loan is into one branch, then I would suggest the name of that branch, e.g. Germanic, Indic, Indo-Iranic, etc. This is, after all, a similar situation as to when we have a cognate set which is confined to a branch, but cannot be reconstructed to the proto-language. When the parallel loan extends across multiple families, as in your example, it would be logical to label it "Indo-European". Might this lead to misapprehensions though?

**Comment by m-scarborough** [comment]
Could someone outline the policy as to what exactly a 'parallel loan' is supposed to be?  I'm beginning to get confused if http://www.cobl.info/cognate/5711/ is a legitimate cognate coding under current policy, because that looks to me like a claim that some putative common ancestor of Iranian and Tocharian borrowed a Sanskrit word at some point in their prehistory, and that cannot be correct.

**Comment by CasFre** [comment]
You are right that this seems awkward and that's exactly what "parallel loan" stands for: cases where we do not assume that there was one common ancestor to all the represented languages into which a lexeme was borrowed, but rather that we assume that several unrelated loan events took place which can't however be pinned down so well as to open up a separate cognate class for each of them.
This most prominently also covers alleged "Kultur-" and "Wanderwörter".

**Comment by cormacanderson** [comment]
See also https://github.com/lingdb/CoBL/wiki/06--Policy-on-Loanwords

**Comment by m-scarborough** [comment]
Ah, ok.  I missed that dispatch.  I understand now, thanks.

**Comment by CasFre** [comment]
Coming back to the original questions about parallel loans:
I must say that, intuitively, I don't like the suggestion to enter "Indo-European" for parallel loans, @cormacanderson. But maybe it is not such a bad idea in the end (as I said myself: in a database, some abstractions in annotation are needed), as the colour boxes tell you what branches are involved and as you also (hopefully) have entries in the loan fields.
What should be the standard entry for root form, then?
I see the following possibilities:

- entering the form of a reference language involved or otherwise the first language in the clade (cf. my algorithm above), accompanied by "≈" (cf. [Policy on Additional Symbols with Root Forms](https://github.com/lingdb/CoBL/wiki/06-Root-Forms-and-Root-Languages).
- entering another dummy sign indicating that a root form in the narrower sense does not exist, e.g. "#".

**Comment by PaulHeggarty** [comment]
As people may have noticed, the latest update to the site has ended up, after discussion between Cormac and myself, choosing the names `Root ref. language` and `Root ref. form` for these fields (so I’ll also rename this issue as such).

**Comment by PaulHeggarty** [comment]
Another discussion has led to a new suggestion, for what to do in instances of Case 3, where a form is found in a branch, but considered probably not to go back to the branch proto-language, as in the case of these two (West Slavic) entries for 'blow':
http://cobl.info/cognate/575/
The danger is that simply entering 'Slavic' for this case is highly ambiguous, and most viewers of the database will not presume that it implies we have an explicit policy to consider that this means we do **not** presume that they go back to the proto-language -- the default interpretation will be the contrary.  So this is a dangerous line to follow and will lead to people being suspicious of our data quality without actually having reason to, but just because we took a non-obvious shortcut in naming policy for our `Root ref. language` field.  

So the new policy in such cases is:
- In the `Root ref. language` field, give the branch but preceded by the word sporadic.  E.g. in the above case:  'sporadic Slavic'.
- In the `Root ref. form` field, the aim would be to enter a 'pseudo-reconstruction' of the form that this cognate set would be expected to have had if it were a genuine proto-form in that branch -- but we need a dedicated symbolic notation to identify that.  The notation proposed is:  !*

Both these points are suggestions so far, so if you feel there is a better term than 'sporadic', or a better symbol than !*, then please say so here.
That is, this would be entered 
to lead to accustion

**Comment by CasFre** [comment]
Well, I would say that our policy on root ref. languages is clear enough and also is in line with general use in historical linguistics: 

- "Proto-[branch]" = can be reconstructed to that stage,

- "[branch]" = cannot be (securely) reconstructed to the respective proto language, but is found in one or more languages of that branch.

Some time ago we said that we would not be willing to introduce sub-branch names as root ref. language names, nor terms such as "Post-PIE" either, so I would say that we decided to have a certain level of abstraction in our annotation there, and Cormac's recent suggestion above to enter "Indo-European" for the parallel loans also points into that direction.
So I would say there is no problem as long as we state this policy clearly for the users.

As far as the root ref. form is concerned, I don't think there is any use in trying to establish pseudo-proto forms if lots of us don't even feel confident in establishing real proto forms for individual language branches. Thus my suggestion would be to use the form of the first lexeme entered in the cognate set (in the example:  fúkať) and combine it with the "vel sim." symbol "≈".
The first lexeme within the cognate set on the individual cognate sets page should, if I'm not mistaken' normally correspond to the lexeme of the reference language for that branch, if this belongs to the cognate set, or is otherwise computed according to the clade structure on the language list, so there would be some consistent mechanism behind it.
This ensures identifiability, gives a coherent picture foor root ref. form and root ref. language, and we don't open doors for accusations on false pseudo proto forms.
And, if you like, you might even add a check-box "not reconstructable" (or whatever) and compute root ref. form and root ref. languages automatically, if this is ticked. ;-)

**Comment by cormacanderson** [comment]
Re the discussion on parallel loans, today I stumbled across an example [from Romance into both Germanic and Slavic languages](http://cobl.info/cognate/5697/), which presents similar issues to @CasFre's example [from Sanskrit into both Iranic languages and Tocharian](http://www.cobl.info/cognate/5711/).

**Comment by cormacanderson** [comment]
A further example, again consisting of Polabian and a number of Germanic languages, can be found [here](http://cobl.info/cognate/5763/).

**Comment by PaulHeggarty** [comment]
I don’t think any individual cognate set merits so much concern, especially because they are loanword cognate sets in any case.  As a practical policy, just separate them into loanword event cognate sets, one for each branch.  The use of 'parallel loan' events is only for practical advantages anyway, of saving time without having to create a separate set for each lexeme.  We can create separate sets for each branch in these very rare cases, and just get on, without worrying too much about root forms.

**Comment by m-scarborough** [comment]
I want to raise here an issue on the policy of root ref. forms based on some odd data I've noticed while I've been working through and checking some cognate sets.  This issue is related to the sources and cleaning-up of the database:

As a matter of general policy, I would like to propose:

As much as possible, when root ref. forms are given that are reconstructed forms, they _should_ have citations to sources in the cognate sets themselves as references for the root ref. form reconstruction.

Now and again I've come across reconstructions that look either unusual or non-standard without justification.  To give one example I've noticed today, THICK in http://cobl.info/cognate/1585/ is currently given as *bʰ(h₂)emg̑ʰ- without references or citation.  This root is more normally reconstructed in standard works (LIV², NIL) as *bʰeng̑ʰ- (as in FAT /cognate/4117/).  

<img width="402" alt="screen shot 2016-10-15 at 17 28 15" src="https://cloud.githubusercontent.com/assets/15671787/19411604/dfb7d1f6-92fc-11e6-8581-4824e019d408.png">

First of all in this case, this should not be the reconstructed form used as the root ref. form in the first place if it is not a commonly accepted reconstruction.  But if we must feel the need to put in non-standard reconstruction in any other case, it should be cited, whether by a published source justifying the non-standard reconstruction (preferred), or by a citation made by the cognacy-coder or editor of the database who entered the form.

I admit that the standardisation of references and root reconstructions for root ref. forms used has been something hard to co-ordinate on since all the cognacy-coders have been working in isolation from each other, and I hope that I'm not being to aggressive in singling out this specific example, but nevertheless, this is exactly the reason why these need to be cited (I have no idea where *bʰ(h₂)emg̑ʰ- comes from) so all cognacy-coders can understand why a specific reconstructed root ref. form is given.  More importantly failure to provide justification for our reconstructed root ref. forms has the potential to reflect poorly on the general quality and reliability of our cognate judgements, and by extension leaves open to criticism any further publications that are based on this database.

**Comment by PaulHeggarty** [comment]
I agree strongly with all of this.  This is why it is so important to fix and improve the whole citation system.  We had a programmers' meeting about this last Thursday, and on Monday the first stage in the new system should be up and running, and we are rushing to get the new system fully working as soon as the programmers can manage it.  Then it will be much faster and more consistent to enter citations.  Thanks for putting this comment up.  Konstantin should shortly be contacting you and putting up a wiki page on the new citation system.

**Comment by RolandPooth** [comment]
even worse: the reference to ALEW I s.v. biñgti given there is a 
mistake, since biñgti has a completely different eytmology.
I agree that such entries are hard to co-ordinate on because all the 
cognacy-coders have been working in isolation from each other;
in my view, this is exactly the reason why these need to meet in a 
single room with a big display and go over these things together someday


Am 15.10.2016 18:54, schrieb Matthew Scarborough:

**Comment by nitsnatsnok** [comment]
I've put this on my task list in #332.
